tensorflow/lite/micro/micro_allocator.cc

```diff
@@ -300,6 +303,9 @@ TfLiteStatus FlatBufferVectorToTfLiteTypeArray(
     *result = const_cast<kTfLiteArrayType*>(
         reinterpret_cast<const kTfLiteArrayType*>(flatbuffer_array));
   } else {
+    // Fixed b/206152717 by removing this check for little endian. This check is
```

Comment:
does this mean some models that work on little endian can not work on big endian? And there is no path to having support for a nullptr flatbuffer_array for big endian?

If that is the case, I would strongly suggest removing big endian support completely since this can be quite confusing.

https://github.com/tensorflow/tflite-micro/issues/304
https://github.com/tensorflow/tflite-micro/issues/455

I'm happy to make a separate PR that removes this support but wanted to first make sure that my understanding is correct as far as big-endian and this current PR is concerned.

Reply:
Yes that is correct. This would only be the case for models with zero-length input / output arrays (those using resource variables or CALL_ONCE).

I agree in principle, but I'm unsure what effects removing big endian support would have.

Reply:
Let's discuss removing big endian in a follow-on CL separately?